src/Analysis/GWASApp/Steps/ConfigureGWAS/ConfigureGWAS.jsx

```diff
@@ -38,6 +39,12 @@ const ConfigureGWAS = ({
   const [showError, setShowError] = useState(false);
   const [jobName, setJobName] = useState('');
   const [errorText, setErrorText] = useState('');
+  const covariatesWithoutCohortNamesAndCohortSizes = covariates.map(
+    ({ cohort_names, cohort_sizes, ...allOtherKeyValuePairs }) => allOtherKeyValuePairs,
+  );
+  const outcomeWithoutCohortNamesAndCohortSizes = Object.fromEntries(
+    Object.entries(outcome).filter(([key]) => key !== 'cohort_names' && key !== 'cohort_sizes'),
+  );
```

Comment:
I would suggest using "workflowCovariates", "workflowOutcome" names or something similar instead of "covariatesWithoutCohortNamesAndCohortSizes" and "outcomeWithoutCohortNamesAndCohortSizes" - that way if we need to filter out something else later, we won't need to change the name of those variables.

Reply:
Variable names updated as requested to: 

```
workflowCovariates
workflowOutcome
```